### PR TITLE
Upgrade kiwigrid/k8s-sidecar:1.12.0

### PR DESCRIPTION
**Pull request by mount986**
# What this PR does / why we need it
The current version is over a year old, and has a critical vulnerability: 
CVE-2020-14343 pyyaml 5.3.1 critical fixed in 5.4

# Which issue this PR fixes
No issue created

# Special notes for your reviewer

# Checklist

- [x] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] CHANGELOG.md was updated

### CODE REVIEW

File: charts/jenkins/values.yaml [hunk 1]
```diff
@@ -311,7 +311,7 @@ controller:
       # jcasc changes will cause a reboot and will only be applied at the subsequent start-up.  Auto-reload uses the
       # http://<jenkins_url>/reload-configuration-as-code endpoint to reapply config when changes to the configScripts are detected.
       enabled: true
-      image: kiwigrid/k8s-sidecar:0.1.275
+      image: kiwigrid/k8s-sidecar:1.12.0
```
Comment (timja) [inline charts/jenkins/values.yaml:314]: changelog and chart version need bumping

Comment (mount986) [reply charts/jenkins/values.yaml:314]: done

**Comment by timja** [comment]
commit history doesn't look right

**Comment by mount986** [comment]
I'm not sure what else you need me to do here..  I'm trying to help out by submitting a pull request rather than just creating an issue and expecting someone else to take care of it, but I don't have the time to jump through hoops here and become an expert at your process..

**Comment by mount986** [comment]
sorry, was a bit out of line.. been kind of a bad day.. if you can help me figure out what I need to do here to correct this and get it pulled, that would be appreciated..

**Comment by timja** [comment]
There's commits from other people appearing in this PR, 

image

I'm guessing your main branch wasn't up to date when you rebased?

**Comment by mount986** [comment]
yeah, when I submitted the chart version and change log, it said I needed to merge the branch.. that's when it added those.. do I need to kill the request and submit again?

**Comment by timja** [comment]
> yeah, when I submitted the chart version and change log, it said I needed to merge the branch.. that's when it added those.. do I need to kill the request and submit again?

probably easiest

**Pull request closed without merging**